pkg/cmd/server/start/start_master.go

```diff
@@ -558,6 +558,7 @@ func startControllers(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) erro
 		kc.RunEndpointController()
 		kc.RunNamespaceController()
 		kc.RunPersistentVolumeClaimBinder()
+		kc.RunPersistentVolumeProvisioner(pvKClient)
```

Comment:
cool, in that case it sounds like a quick copy/paste of the existing role, removing the extra permissions it doesn't need, and using that new SA/role to get the client.